config/webpack/custom.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
```

Comment:
is dit ding echt nodig? Wat is die `jquery/src/jquery`?

Reply:
Dat is het relatieve pad naar jquery.js onder node_modules. Volgens mij is het een of andere hack dattie jquery anders niet kan vinden. Alle node_modules hebben namelijk een index.js in hun root directory, maar jquery heeft dat om een of andere reden niet. Ik zal kijken of het problemen geeft als ik het weghaal.